app/App/Panel/Main/Account/Requests/TransactionRequest/TxOverlay/TxFeeOverlay/index.js

```diff
@@ -0,0 +1,348 @@
+import React from 'react'
+import Restore from 'react-restore'
+// import utils from 'web3-utils'
+
+import svg from '../../../../../../../../../resources/svg'
+import link from '../../../../../../../../../resources/link'
+import { gweiToWei, gweiToWeiHex } from '../../../../../../../../../resources/utils'
+
+import BigNumber from 'bignumber.js'
+
+
+//  <div className='txModuleTop'>
+// <div className={'txModuleTopData txModuleTopDataExpanded'}>
+  // <div className='transactionDataNotice'>{svg.octicon('issue-opened', { height: 26 })}</div>
+  // <div className='transactionDataLabel'>View Data</div>
+  // <div className='transactionDataIndicator' onMouseDown={() => this.copyData(req.data.data)}>
+    // {svg.octicon('clippy', { height: 20 })}
+  // </div>
+// </div> 
+// <div className='txModuleBody'>
+
+const FEE_MAX_TOTAL_ETH_WEI = 2 * 1e18
+
+class TxFeeOverlay extends React.Component {
+  constructor (props, context) {
+    super(props, context)
+    
+    this.moduleRef = React.createRef()
```

Comment:
what does this do?

Reply:
This creates a references to an actual DOM node